websocketpp/transport/asio/endpoint.hpp

```diff
@@ -518,17 +518,15 @@ class endpoint : public config::socket_type {
         lib::error_code & ec)
     {
         using lib::asio::ip::tcp;
-        tcp::resolver r(*m_io_service);
-        tcp::resolver::query query(host, service);
-        tcp::resolver::iterator endpoint_iterator = r.resolve(query);
-        tcp::resolver::iterator end;
-        if (endpoint_iterator == end) {
+        tcp::resolver r(*m_io_context);
+        tcp::resolver::results_type endpoints = r.resolve(host, service);
```

Comment:
Suggest naming the variable something like `results` instead of `endpoints` - as they are of type `results_type` (and "results" seems to be the asio/boost way to refer to them)

Reply:
Additionally, I'd suggest updating the comment for this function since it refers to "Asio documentation for ip::basic_resolver_query::basic_resovler_query's constructors"...which we aren't using.  I'd direct them instead to the "Asio documentation for the ip::basic_resolver::resolve function". There are two places in comments where it references the `basic_resolver_query` constructor (the two `listen` functions)